### PR TITLE
Updated CHANGELOG preparing v0.8.0 release

### PR DESCRIPTION
**Pull request by rporres**
Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

**Review by meyskens — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-## 0.7.1 (Unreleased)
+## 0.8.1 (Unreleased)
```
Comment (alexsomesan) [inline CHANGELOG.md]: This line needs to be actual version to be released next. 
The release tooling then timestamps it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+## 0.8.0 (Feb 11, 2019)
```
Comment (alexsomesan) [inline CHANGELOG.md]: This line is not needed.
The release tooling will insert the timestamp in place of the `Unreleased` word on the first line.

**Review by alexsomesan — COMMENTED** [review]
Looks good, the version line needs a different format. See my spot comments.
Let me know if you want me to push a correction commit instead, if that's easier.

**Comment by rporres** [comment]
Fixed, @alexsomesan. I hope I've understood correctly 😅

**Review by alexsomesan — APPROVED** [review]
Looks great! 👍 

Don't stress about, it's confusing. I learned the hard way, I messed up the first release I had to do for the same reason.

**Comment by alexsomesan** [comment]
Release build is running....

**Comment by alexsomesan** [comment]
Aaaand it's out: https://releases.hashicorp.com/terraform-provider-helm/0.8.0/

**Pull request merged**